workflow.yml

```diff
@@ -0,0 +1,74 @@
+openapi: 3.0.0
+info:
+  version: "0.0.3-unstable"
+  title: VC Workflow HTTP API
+  description: This is an Experimental Open API Specification for the [VC Data Model](https://www.w3.org/TR/vc-data-model/).
+  license:
+    name: W3C Software and Document License
+    url: http://www.w3.org/Consortium/Legal/copyright-software.
+  contact:
+    name: GitHub Source Code
+    url: https://github.com/w3c-ccg/vc-api
+paths:
+  /workflows/{name}/start:
+    post:
+      summary: Start a workflow.
+      description:
+        Starts a workflow with the given name. Optionally, additional information can be provided to the server in the request body that can help it fine tune what information to request from the client in order to continue with the workflow.
+      requestBody:
+        description: Optional and arbitrary POST data that the client might provide to the server to help it fine tune what information to request from the client in order to continue with the workflow.
+        content:
+          application/json:
+            schema:
+              oneOf:
+                - type: object
+                - type: string
+      responses:
+        "201":
+          description: A new workflow was successfully created for the client.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/WorkflowStartResponse"
+        "400":
+          description: The workflow request was malformed.
+        "500":
+          description: Internal Error
+        "501":
+          description: Not implemented
+  /workflows/{id}/presentations:
+    post:
+      summary: Present information to an active workflow.
+      operationId: storePresentation
```

Comment:
abuse of existing operationId... see https://github.com/w3c-ccg/vc-api/blob/main/holder.yml#L88


perhaps more evidence that these APIs already exist in some form?

Reply:
Good catch, wasn't meaning to do that. Fixed in 95535e63b44331b4af5fe33f34ea39e37c709544.